src/Angular.js

```diff
@@ -583,6 +582,9 @@ function isPromiseLike(obj) {
   return obj && isFunction(obj.then);
 }
 
+function isTypedArray(value) {
+  return /^\[object (Uint8(Clamped)?)|(Uint16)|(Uint32)|(Int8)|(Int16)|(Int32)|(Float(32)|(64))Array\]$/.test(toString.call(value));
```

Comment:
It's not going to matter to code-gen (at least in v8), but we should probably declare the regexp as a variable outside of the function, so that it's more readable